grafana/grafana/base/jupyterhub-sli-dashboard.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: odh-jupyterhub-sli
+  labels:
+    app: grafana
+spec:
+  url: https://raw.githubusercontent.com/odh-manifests/grafana/grafana/base/dashboards/jupyterhub-sli-slo.json
```

Comment:
So the idea of having the jsons for ODH related component dashboards stored in a separate folder in `odh-manifests` was that if these dashboards are "basic/generic" enough, then we (operate-first) can point to these jsons as a remote URL in our grafana resource and if we need to customize the dashboard further for our needs, we can take the json and store the modified jsons in our SRE repo.

Reply:
I think the reason for keeping the dashboard jsons separate was because some installations like IDH were not using the Grafana Operator and they were planning to use them directly, @anishasthana and @accorvin might know better.

I think we can put the jsons in a configmap and reference them in the `GrafanaDashboard` object ([doc](https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md#config-map-references)), this way IDH should be able to directly reference the dashboard.

Reply:
++ for keeping the jsons separate. That makes life a lot easier for us downstream.

Reply:
After an offline conversation with @hemajv explaining the reasoning for separating the json from the `GrafanaDashboard`, I understand now. 

We need to use a tagged reference to this dashboard url and, as far as I know, we can't do that within this PR since it is adding the files we need to "tag".  From @4n4nd's comment, we should use a `ConfigMap` reference instead of the url so that operate-first, downstream, ... can override it easily

Reply:
shall I go ahead and update the PR to have the dashboard jsons embedded in the `ConfigMap` then rather than saving the individual json files? @anishasthana wdyt?

Reply:
Hmmm, configmap generator + JSONS are easier to work with as developers. indented jsons in yaml are pretty miserable -- any cahnce we could do that instead, @LaVLaS ? If not, I'm fine with embedding them directly in the CM.

Reply:
I have no issue with the ConfigMapGenerator referencing pure json